bin/db_prep

```diff
@@ -19,26 +19,32 @@ end
 
 Dir.chdir APP_ROOT do
   puts "\n== Preparing Database =="
-  run_or_exit 'bundle exec rake db:create'
-  run_or_exit 'bundle exec rake db:structure:conditional_load'
-  run_or_exit 'bundle exec rake db:migrate'
+  run_or_exit 'bundle exec rake db:create:primary'
+  run_or_exit 'bundle exec rake db:schema:conditional_load'
```

Comment:
Are we intentionally going back to schema (or is there a different meaning in Rails 6?)

Reply:
Oh, I see the comment below (structure:load is deprecated)
note for later, we're going to need to figure out how to get some of the more complicated things we do into gems or other means so schema:load recognizes them.

Reply:
Its meaning has changed. it does sql or schema (ruby) based on what you set it to in application.rb

Reply:
Cool, so if we just tell application.rb to do sql for all the dbs, we can go back to schema:load everywhere?

Reply:
I only know of one place to set it, and I think it applies to all the databases. So, I think schema:load everywhere or the conditional version of that.